fractal_server/tasks/v2/background_operations_ssh.py

```diff
@@ -316,9 +316,19 @@ def background_collect_pip_ssh(
                     python_bin=Path(python_bin),
                 )
 
+                # Prepare some task-group attributes
+                task_group_attrs = dict(
+                    pkg_name=task_pkg.package_name,
+                    version=task_pkg.package_version,
+                )
+                if task_pkg.is_local_package:
+                    task_group_attrs["origin"] = "wheel-file"
```

Comment:
please open an issue about this missing test

Reply:
https://github.com/fractal-analytics-platform/fractal-server/issues/1864